CHANGELOG.md

```diff
@@ -180,11 +187,11 @@
 * Update to Apache Server Configs 1.1.0.
 * Add `initial-scale=1` to the viewport `meta`
   ([#1398](https://github.com/h5bp/html5-boilerplate/pull/1398)).
-* Vertical centering for audio-, canvas- and video-tags
+* Vertical centering for audio-, canvas* and video-tags
```

Comment:
we need to revert the changed `-` to `*` in some places - like here.

Reply:
👍 I missed a few